Mailbox.Tests/DataLoaderTests.cs

```diff
@@ -0,0 +1,96 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System;
+
+namespace Mailbox.Tests
+{
+    [TestClass]
+    public class DataLoaderTests
+    {
+        [TestMethod]
+        public void DataLoader_LoadMailbox_Success()
+        {
+            string fileName = Path.GetRandomFileName();
+            Stream fileStream = File.Open(fileName, FileMode.Create);
```

Comment:
Stream implements IDisposable and should be cleaned up.
```suggestion
            using Stream fileStream = File.Open(fileName, FileMode.Create);
```